ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-947] Add partner premium bonds page

## Description of change
Add partner premium bonds page
Add to serializer, struct, validator and CYAs

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-947

## Notes for reviewer
This does not cover changing Partner from YES to NO - resetting etc needs to be done as a wider piece.

## Screenshots of changes (if applicable)

![image](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/d3a7079a-b9af-4e86-8743-d9f77f1f24c7)
![image](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/e4dbfdac-ff88-489d-ba6b-bc3237c91cc8)

## How to manually test the feature

Start an application,
Case type Indictable
Partner Yes
Income £12475 Yes
Continue to Capital
Check errors when nothing selected
Check errors when Yes selected and no numbers input
Complete page, check shows correctly on Capital CYA
Complete App, check shows correctly on app CYA
